x-pack/filebeat/input/httpjson/input_test.go

```diff
@@ -1357,9 +1357,9 @@ var testCases = []struct {
   </item>
 </order>
 `
-				io.ReadAll(r.Body)
+				_, _ = io.ReadAll(r.Body)
 				r.Body.Close()
-				w.Write([]byte(text))
+				_, _ = w.Write([]byte(text))
```

Comment:
Please revert this.

Reply:
The old code doesn't pass the linter. This doesn't change the existing behavior of the unit test.

Reply:
The linter does not block merging.